build.gradle

```diff
@@ -61,7 +61,7 @@ ext {
     buildToolsVersion = '25.0.2'
 
     // App dependencies
-    supportLibraryVersion = '25.1.0'
+    supportLibraryVersion = '25.2.0'
```

Comment:
Is support lib version upgrade related to vectors?
Should we even upgrade to later version `25.3.1`?